chainer/function.py

```diff
@@ -94,7 +94,8 @@ class FunctionAdapter(function_node.FunctionNode):
        object changes the reference to a weak one.
 
     Args:
-        function (Function): The function object to wrap.
+        function (~chainer.Function):\
```

Comment:
Avoid line continuation with `\`. Instead, indent the following line correctly. Applied to other instances.